Script/char.dat

```diff
@@ -4817,13 +4822,14 @@ gibberling
   CanBeGenerated = true;
   Sex = UNDEFINED;
   CanUseEquipment = false;
-  CanTalk = false;
+  CanTalk = false; /* can, but no-one understands the gibberish */
```

Comment:
Would be funny to have it spew random "words" when spoken to

Reply:
I'll write down a note to make gibberlings shout out random gibberish continuously, similarly to kamikaze dwarves' singing.